src/e2e/java/teammates/e2e/cases/lnp/FeedbackSessionSubmitLNPTest.java

```diff
@@ -0,0 +1,273 @@
+                DataBundle dataBundle = loadDataBundle(getJsonDataPath());
+                List<List<String>> csvData = new ArrayList<>();
+
+                dataBundle.students.forEach((key, student) -> {
+                    List<String> csvRow = new ArrayList<>();
+
+                    csvRow.add(student.googleId); // "googleId" is used for logging in, not "email"
+                    csvRow.add("no");
+                    csvRow.add(student.googleId);
+                    csvRow.add(COURSE_ID);
+                    csvRow.add(FEEDBACK_SESSION_NAME);
+                    csvRow.add(student.email);
+
+                    csvData.add(csvRow);
+
+                });
+
+                return csvData;
+            }
+        };
+    }
+
+    private Map<String, String> getRequestHeaders() {
+        Map<String, String> headers = new LinkedHashMap<>();
+
+        headers.put("X-CSRF-TOKEN", "${csrfToken}");
+        headers.put("Content-Type", "application/json");
+
+        return headers;
+    }
+
+    @Override
+    protected ListedHashTree getLnpTestPlan() {
+        ListedHashTree testPlan = new ListedHashTree(JMeterElements.testPlan());
+        HashTree threadGroup = testPlan.add(
+                JMeterElements.threadGroup(NUMBER_OF_USER_ACCOUNTS, RAMP_UP_PERIOD, 1));
+        threadGroup.add(JMeterElements.csvDataSet(getPathToTestDataFile(getCsvConfigPath())));
+        threadGroup.add(JMeterElements.cookieManager());
+        threadGroup.add(JMeterElements.defaultSampler());
+        threadGroup.add(JMeterElements.onceOnlyController())
+                .add(JMeterElements.loginSampler())
+                .add(JMeterElements.csrfExtractor("csrfToken"));
+
+        HeaderManager headerManager = JMeterElements.headerManager(getRequestHeaders());
+        threadGroup.add(headerManager);
+
+        //Backdoor to retrieve questionId which is generated only on backend server
+        String fqId = BackDoor.getFeedbackQuestionId(COURSE_ID, FEEDBACK_SESSION_NAME, 1);
+
+        String body = "{\"questionType\": \"TEXT\","
+                + "\"recipientIdentifier\": \"${studentEmail}\","
+                + "\"responseDetails\": {\"answer\": \"<p>test</p>\", \"questionType\": \"TEXT\"}}";
+        String fourthPath = "webapi/response?questionid=" + fqId
+                                    + "&intent=STUDENT_SUBMISSION";
+        threadGroup.add(JMeterElements.httpSampler(fourthPath, POST, body));
+
+        return testPlan;
+    }
+
+    @BeforeClass
+    public void classSetup() {
+        createTestData();
+        persistTestData();
+    }
+
+    @Test
+    public void runLnpTest() throws IOException {
+        try {
+            Thread.sleep(3000);
+        } catch (Exception e) {
+            System.out.println("Error while executing LnP Test.");
+            System.out.println(e.toString());
+        }
+        runJmeter(true);
+    }
+
+    /**
+     * Removes the entities added for the instructors' student enrollment L&P test.
```

Comment:
Remember to change the doc

Reply:
Updated javadoc to reflect on the actual test.